README.rst

```diff
@@ -91,9 +92,35 @@ Getting Help
 Documentation
 =============
 
-This is a new service, and is a work in progress (as most things are)! There is no official home for this app's technical docs just yet, but there could be soon. Please reach out to someone on the Purchase Squad if you have questions.
+How to use this service
+------------------------------------------------
 
-(TODO: `Set up documentation <https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs>`_)
+The sanctions endpoint will check users against the SDN API and return a hit count - if there is a match found, a record in the sanctions database (SanctionsCheckFailure) will be created.
+
+Example of making a POST request to the `api/v1/sdn-check/` endpoint:
+
+.. code-block::
+
+  response = self.client.post(
+      'https://sanctions.edx.org/api/v1/sdn-check/',
+      timeout=settings.SANCTIONS_CLIENT_TIMEOUT,
+      json={
+          'lms_user_id': user.lms_user_id, # optional
+          'username': user.username, # optional
+          'full_name': full_name,
+          'city': city,
+          'country': country,
+          'metadata': { # optional
+              'order_identifer': 'EDX-123456',
+              'purchase_type': 'program',
+              'order_total': '989.00'
+          },
+          'system_identifier': 'commerce-coordinator', # optional
+          'sdn_api_list': 'ISN,SDN', # optional, default is 'ISN,SDN'
```

Comment:
thinking about this... we actually shouldn't let callers change this value for this endpoint, if the specific end point is for SDN checking. ><' -- we can still record `ISN,SDN` in the DB table, but no other values here make sense with this end point, so we probably shouldn't let people override it

Reply:
This is copy/paste from what was done in the subscription version - https://github.com/edx/ecommerce/blob/6b88537d3bc5aba39acd8671244932bbae8c574d/ecommerce/extensions/payment/views/sdn.py#L121. Do you remember why/if there was a reason?

For the other version in ecommerce, it's retrieved from site configuration - https://github.com/edx/ecommerce/blob/59efeb07c85423747f38b52bd3c52749c95d7f4f/ecommerce/extensions/payment/core/sdn.py#L42

Reply:
I think I probably did it for flexibility, partially begotten of me not being entirely familiar with the space (as in, dont be too opinionated if I don't have a reason for it).

my thinking: in the context of a service that has that flexibility built in to the model, we do want to be more specific in this endpoint, so like, would we ever have a non "sdn,isn" value for an SDN endpoint? my guess is no, because if it was something else, we'd probably have a different endpoint, and therefore should not give the user the option to put something different

Reply:
If you don't love it for being hardcoded in the file itself and would prefer to put it in a settings file, that's also fine, and probably "better". I don't feel strongly about it either way

Reply:
Unless there is no need to use the ISN list as well. But yes, I prefer this to be in settings 👍